widget/button_test.go

```diff
@@ -65,6 +65,10 @@ func TestButton_Tapped(t *testing.T) {
 }
 
 func TestButton_Disable(t *testing.T) {
+	app := test.NewApp()
+	defer test.NewApp()
+	app.Settings().SetTheme(theme.LightTheme())
```

Comment:
This is not related to fixing #1663, please remove

Reply:
Are you sure? It seemed to start failing when I added the tree test, hence the commit message "Resolve flakey button test impacted by this tree change".

Reply:
Seems like this was broken on release branch and we didn't notice.
Rebase on top of the fix, so it's no longer part of this PR diff.